packages/data-broker/src/services/dhis/DhisService.js

```diff
@@ -158,100 +153,103 @@ export class DhisService extends Service {
     return events;
   };
 
-  fetchEventAnalyticsForPrograms = async (api, programCodes, query) => {
-    const allHeaders = [];
-    let metaData = { items: {}, dimensions: {} };
-    let width = 0;
-    let height = 0;
-    const rows = [];
+  translateDhisEventAnalytics = async (dhisAnalytics, dataSources) => {
+    const dataElementCodes = dataSources.map(({ code }) => code);
 
-    const fetchAnalyticsForProgram = async programCode => {
```

Comment:
This method had been inlined, because the merging has been extracted into another method